lib/postgrex/protocol.ex

```diff
@@ -654,18 +675,28 @@ defmodule Postgrex.Protocol do
     end
   end
 
+  defp execute_send( s, status, %Stream{ state: :suspended, portal: portal, max_rows: max_rows, query: query } = stream, _params, buffer ) do
+    %Query{ name: name } = query
+    messages = [ msg_execute(name_port: portal, max_rows: max_rows) | sync_msgs( status, name ) ]
+    send_and_recv( s, status, stream, buffer, messages, &execute_recv/4 )
```

Comment:
It was because the code passes arguments and an anonymous fun to a function and doesn't use all of the arguments in the function except to call the anonymous fun. It would be clearer to copy the explicit control flow used elsewhere or pass an anonymous fun with unused arguments already completed in fun, e.g. &fun(&1, arg) instead of fun/2.